frameworks/engineering/mobile.md

```diff
@@ -172,6 +172,7 @@ topics:
           - "Builds simple screens or flows"
           - "Asks questions and actions feedback from PRs and other engineers"
           - "Fixes simple bugs or issues"
+          - "Produces well-polished interfaces in collaboration with a designer"
```

Comment:
`well-polished` might need more definition, it's a little up to interpretation of engineers + managers.